save-api/README.md

```diff
@@ -42,10 +42,10 @@ required fields from another section could have empty values.
 Current API required few cli arguments for execution:
 ```
   -m execution mode: git or standard
-  -u username in SAVE-cloud system
-  -t OAuth token for SAVE-cloud system
+  -u user information (source, i.e. `github` and username, separated by @) in SAVE-cloud system
+  -t token, generated by SAVE-cloud system
 ```
 
 Example:
 
-    ./gradlew save-api:run -DincludeSaveApi='true' --args='-u user -p password -m git'
\ No newline at end of file
+    ./gradlew save-api:run -DincludeSaveApi='true' --args='-u github@user -t token -m git'
```

Comment:
We need to pass username and source into http client configuration via basic auth, however as you can remember http basic have own style for it `Basic name:password`, so if we will use `:` as separator, http client will recognize our data `-u github:user -t token` as `Basic name = github, password = user:token`, since it will split it after first `:`

Reply:
Then, maybe, introduce a new CLI argument `--oauth2Source`?

Reply:
Good idea, added